app/assets/yge/youvegoteth/send.js

```diff
@@ -43,40 +44,54 @@ var updateEstimate = function(e) {
 
 };
 
-window.onload = function() {
-  jQuery('#amount').on('keyup blur change', updateEstimate);
-  jQuery('#token').on('change', updateEstimate);
-
-  unPackAddresses();
-
-  var min_send_amt_wei = 6000000;
+function isSendTipPage() {
+  return (/\/tip\/send/).test(window.location.pathname);
+}
 
+function prePopulateFounderFields() {
```

Comment:
I am not happy with the function name I came up with. Feel free to suggest a better alternative.

Reply:
lol my bad :see_no_evil: 

Fixed.